content/en/blog/_posts/2024-03-11-diy-create-your-own-cloud-with-kubernetes-part-3/index.md

```diff
@@ -0,0 +1,160 @@
+---
+layout: blog
+title: "DIY: Create Your Own Cloud with Kubernetes (Part 3)"
+slug: diy-create-your-own-cloud-with-kubernetes-part-3
+date: 2024-03-11
+---
+
+**Author**: Andrei Kvapil (Ænix)
+
+Here we are, approaching the most interesting part - running Kubernetes in Kubernetes. In this article, I'll show you technologies such as Kamaji and Cluster API, as well as how to integrate them with KubeVirt.
+
+In previous articles, we already talked about [how we prepare Kubernetes on bare metal](/blog/2024/01/22/diy-create-your-own-cloud-with-kubernetes-part-1/), and [how to turn Kubernetes into virtual machines management system](/blog/2024/01/22/diy-create-your-own-cloud-with-kubernetes-part-2). This article concludes the series by explaining how, using all of the above, you can build a full-fledged managed Kubernetes and run virtual Kubernetes clusters with just a click.
```

Comment:
I've not time for a longer review just now, but: avoid using “we” unless it means _[this article has multiple authors, and they collectively &hellip;]_

Reply:
fixed, thanks for review